eidpsam/src/test/java/uk/ac/ucl/eidp/auth/jwt/JwtTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.ac.ucl.eidp.auth.jwt;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+import static org.testng.Assert.assertTrue;
+
+import org.testng.annotations.Test;
+
+import java.nio.charset.Charset;
+import java.security.Key;
+import java.text.ParseException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ * Unit tests for Jwt.
+ * @author David Guzman d.guzman at ucl.ac.uk
+ */
+public class JwtTest {
+  
+  private final String apiKeySecret = "lIySuazq3y2dCKaJhmJyuA==";
+  private final String credentialsEncoding = "UTF-8";
```

Comment:
\[FindBugs\] INFO: SS: Unread field: uk\.ac\.ucl\.eidp\.auth\.jwt\.JwtTest\.credentialsEncoding; should this field be static?